README.md

```diff
@@ -48,61 +62,48 @@ These are the branches in use:
+2. Ideas that are "approved in principle" are eligible for a new RFC in the [F# language design repo](https://github.com/fsharp/fslang-design). This is where the technical specification and discussion of approved suggestions go.
+3. Implementations and testing of an RFC are submitted to this repository.
 
-This project is subject to the MIT License. A copy of this license can be found in [License.txt](License.txt) at the root of this repo.
+## Additional project documentation
 
+The following links can help you get an overview of some technical aspects of the F# language and compiler:
 
-## Using
-
-For typical installs of  F#, see http://fsharp.org.
-
-### Using Nightly Releases of Visual F# Tools
-
-To setup Visual Studio to use the latest nightly releases of the Visual F# Tools:
-
-https://blogs.msdn.microsoft.com/dotnet/2017/03/14/announcing-nightly-releases-for-the-visual-f-tools/
-
-If you wish to set up a *Preview* nightly atop Visual Studio preview builds, you can either [download a VSIX Manually from here](https://dotnet.myget.org/feed/fsharp-preview/package/vsix/VisualFSharp) or set up a VSIX feed in visual studio from [here](https://dotnet.myget.org/F/fsharp-preview/vsix).
-
-### Using CI Builds
+* [The F# Compiler Technical Guide](http://fsharp.github.io/2015/09/29/fsharp-compiler-guide.html) 
+* [The F# Language Specification](http://fsharp.org/specs/language-spec/)
 
-To install F#, see http://fsharp.org.
+## License
 
-To download the bits for the latest CI builds see [these instructions](https://github.com/Microsoft/visualfsharp/wiki/Using-CI-Builds). This includes and ZIPs containing the F# compiler and VSIX installers for the Visual F# IDE Tools.
+This project is subject to the MIT License. A copy of this license is in [License.txt](License.txt).
 
-### Using F# on a build server or computer without an F# installation
+## Code of Conduct
 
-If you wish to use the latest F# compiler on a computer without Visual Studio 2017 installed, you can add the nuget package ``FSharp.Compiler.Tools`` to your projects. This will replace the in-box compiler with the version contained in the package.
-The actual package is built in https://github.com/fsharp/fsharp.
+This project has adopts the [Contributor Covenant](http://contributor-covenant.org/) code of conduct to clarify expected behavior in our community. You can read it at [CODE_OF_CONDUCT](CODE_OF_CONDUCT.md).
 
-You will need to adjust the targets reference on your project file to use the targets file from the installed ``FSharp.Compiler.Tools`` package.
-See https://github.com/fsharp/fsharp/issues/676 for how to modify your project file.
+## Get In Touch
 
-## Code of Conduct
+Members of the [F# Software Foundation](http://fsharp.org) are invited to the [FSSF Slack](http://fsharp.org/guides/slack/). The `#compiler` and `#editor-support` channels are where other contributors typically chat with others.
```

Comment:
I think the last sentence here is worded a tad strange. Maybe something like:

"You can find support from other contributors in the `#compiler` and `#editor-support` channels."